rs/rust_canisters/canister_test/src/canister.rs

```diff
@@ -1119,8 +1119,6 @@ impl<'a> Install<'a> {
             canister.canister_id,
             self.wasm.0.clone(),
             payload,
-            self.compute_allocation,
-            self.memory_allocation,
```

Comment:
Can you also delete these fields from Install for us? Or if that is too painful, leave a comment that says they have no operational effect, and that we should delete them ourselves at some point?

Reply:
I would prefer to not have to tackle specifics in your parts if I don't have to. Where would you like for me to add a comment about these fields have no effect? From a quick search I could really find such fields in some internal types you use (probably I don't know the right places to check).

Reply:
I think you would add such a comment on line 923 of this file.

Reply:
Done.